pkg/cmd/server/start/admission_sync_test.go

```diff
@@ -16,6 +16,7 @@ import (
 var admissionPluginsNotUsedByKube = sets.NewString(
 	"AlwaysAdmit",            // from kube, no need for this by default
 	"AlwaysDeny",             // from kube, no need for this by default
+	"AlwaysPullImages",       // from kube, not enabled by default
```

Comment:
We do not.  It's only important in some environments.  Agree with David to add detail.